kdb/key.go

```diff
@@ -223,7 +223,7 @@ func (k *CKey) Bytes() []byte {
 }
 
 // String returns the string value of the Key.
-func (k *CKey) String() string {
+func (k *CKey) Value() string {
```

Comment:
I don't think it's a good idea. As you can see, it calls `keyString`. There is also `keyValue`, do what you propose here could be exceedingly confusing.

```suggestion
func (k *CKey) StringValue() string {
```

or 

```suggestion
func (k *CKey) ValueString() string {
```

would be acceptable IMO, however, as stated in #19 I think we should wait for ElektraInitiative/libelektra#4853 before changing the Go API.

Reply:
ok, I will then revert this change.

See comment: https://github.com/ElektraInitiative/go-elektra/issues/19#issuecomment-1464892449